### ISSUE TITLE
Loss in neural net

### ISSUE THREAD

**Issue by StrawHatDrag0n** [machinelearningnanodegree/stanford-cs231#5]
I cannot get the training loss below 0.2

**Comment by kvn219** [comment]
Hi StrawHatDragon100!

Which notebook or code are you referring too?
